Lib9c.Policy/Policy/BlockPolicySource.cs

```diff
@@ -72,6 +72,12 @@ public IBlockPolicy GetPolicy(Planet planet)
                     maxTransactionsPerBlockPolicy: MaxTransactionsPerBlockPolicy.Heimdall,
                     maxTransactionsPerSignerPerBlockPolicy: MaxTransactionsPerSignerPerBlockPolicy.Heimdall
                 ),
+                Planet.HeimdallInternal => GetPolicy(
+                    maxTransactionsBytesPolicy: MaxTransactionsBytesPolicy.Heimdall,
+                    minTransactionsPerBlockPolicy: MinTransactionsPerBlockPolicy.Heimdall,
+                    maxTransactionsPerBlockPolicy: MaxTransactionsPerBlockPolicy.Heimdall,
+                    maxTransactionsPerSignerPerBlockPolicy: MaxTransactionsPerSignerPerBlockPolicy.HeimdallInternal
```

Comment:
Why is it different with the Heimdall network? If the `maxTransactionsPerSignerPerBlockPolicy` is different with mainnet, a case when a service having not enough MEAD produces many txs without enough interval and their txs are failed, cannot be caught in internal testing. (If internal's `maxTransactionsPerSignerPerBlockPolicy` is bigger than mainnet's one.)